### ISSUE TITLE
LSP Progress never completes when `notification.filter`  is set to `WARN`

### ISSUE THREAD

**Issue by etrnal70** [j-hui/fidget.nvim#217]
Since notifications and LSP seems unrelated, this is likely a bug

Current config is
```lua
      {
        "j-hui/fidget.nvim",
        opts = {
          progress = {
            ignore_done_already = true,
            ignore = { "null-ls" },
            display = {
              progress_ttl = 25,
              render_limit = 5,
            },
          },
          notification = {
            filter = vim.log.levels.WARN,
            override_vim_notify = true,
            window = { winblend = 0, max_width = 75 },
          },
        },
      },
```

If i comment out the filter, LSP progress will be completed

**Comment by j-hui** [comment]
Oops, I think this is a logic bug with how I've set up the default log levels for LSP progress messages. I'll set them `WARN` in the next release.

**Comment by j-hui** [comment]
Should be fixed by [ebb8e44](https://github.com/j-hui/fidget.nvim/commit/ebb8e44d6c37337e3b4f9bce31842573fa96bf8d)
